DuckDuckGo/TabViewController.swift

```diff
@@ -1022,10 +1020,42 @@ extension TabViewController: WKNavigationDelegate {
         detectedNewNavigation()
         checkLoginDetectionAfterNavigation()
     }
+    
+    private func requestForDoNotSell(basedOn incomingRequest: URLRequest) -> URLRequest? {
+        // For now, only enable for select sites, due to an issue with back navigation this can introduce
```

Comment:
I think someone mentioned in the Asana comments we should call this out explicitly in a comment. Anything else you think we should add to it?